dashboard/db/migrate/20170331235243_create_experiments.rb

```diff
@@ -0,0 +1,16 @@
+class CreateExperiments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :experiments do |t|
+      t.string :name
+      t.string :type
+      t.datetime :start_time
+      t.datetime :end_time
+      t.integer :section_id
+      t.integer :percentage
+      t.datetime :earliest_section_start
+      t.datetime :latest_section_start
```

Comment:
Any reason not to make explicit which of these fields are optional vs. required? Which of these fields have default values?